src/std/srfi/159/pretty.ss

```diff
@@ -0,0 +1,14 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; SRFI-159: pretty printing format combinator
+
+(import ../1
+        ../125
+        ../130
+        ./base
+        ./show
+        ./string)
+(export
+  pretty pretty-shared pretty-simply
+  joined/shares try-fitted)
```

Comment:
(since I'm doing some work with srfi-159 and happen to have an eye on gerbil) these two are not part of srfi-159 so you may want to exclude them together with pretty-shared too.

Reply:
@ghost can you address this before we merge?